aiidalab/app.py

```diff
@@ -861,8 +863,8 @@ def update_app(
             self.refresh()
             return version
 
-    def uninstall_app(self, _=None):
-        """Perfrom app uninstall."""
+    def uninstall_app(self, _: Any = None) -> None:
+        """Uninstall application."""
```

Comment:
This does not related to this PR, but why do we add a parameter which is not used?

Reply:
Good point, the only instance where I found this function used is in `aiidalab-home`, and it doesn't pass any arguments. 

https://github.com/aiidalab/aiidalab-home/blob/2b6ea6d68bcd2edcbb6fb8689338fc721d06c360/home/app_manager.py#L510

It's possibly a historical artifact when this function was called as part of the traitlets / ipywidgets callback. I'll remove it.